weepickle/test/src/com/rallyhealth/weepickle/v1/JsonPointerVisitorTests.scala

```diff
@@ -35,7 +35,7 @@ object JsonPointerVisitorTests extends TestSuite {
       test - assertPathFailure("""{"foo": ["", -666], "s": "", "i": 5, "b": true}""", "/foo/1")
       test - assertPathFailure("""{"foo": ["", -666, ""], "s": "", "i": 5, "b": true}""", "/foo/1")
       test - assertPathFailure("""{"foo": [], "s": -666, "i": 5, "b": true}""", "/s")
-      test - assertPathFailure("""{"foo": [], "s": "", "i": "-666", "b": true}""", "/i")
+      test - assertPathFailure("""{"foo": [], "s": "", "i": "-666NAN", "b": true}""", "/i")
```

Comment:
What is this?!

Reply:
because I changed DefaultTos for int to allow visitString, then this thing that used to fail now succeeds ("-666".toInt == -666). So "-666NAN" makes sure it actually fails ("-666NAN".toInt *boom*).

Reply:
Got it, thanks!